homeassistant/components/enocean/__init__.py

```diff
@@ -1,6 +1,9 @@
 """Support for EnOcean devices."""
+import logging
+
 import voluptuous as vol
 
+from homeassistant.components.enocean import services
```

Comment:
hi @flexxor 👋
As this PR will most likely never be merged to the repo like all the others, I was implementing it in my custom integration available [here](https://github.com/RomeHein/homeassistant_enocean), but when running it on HA, the import of this line does not work. I'm absolutely not familiar with HA code base and infra, so could you assist on that part ? 
This will help a lot of people on french forums who are waiting for this feature for many years. 🙏

Reply:
Ok, in order to make it work, I had to change the import like so:
```suggestion
from .services import async_setup_services
```